commit 9f95dc5d227c5a155d39208dfd15aacaa509b59b
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Mar 8 11:26:57 2015 +0100

    hurd: Make libc able to call pthread stubs
    
    	* sysdeps/mach/hurd/bits/libc-lock.h [_LIBC]: Include
    	<bits/libc-lockP.h>

diff --git a/sysdeps/mach/hurd/bits/libc-lock.h b/sysdeps/mach/hurd/bits/libc-lock.h
index 24c3aa8..63d3e81 100644
--- a/sysdeps/mach/hurd/bits/libc-lock.h
+++ b/sysdeps/mach/hurd/bits/libc-lock.h
@@ -203,4 +203,10 @@ void *__libc_getspecific (__libc_key_t key);
 
 #endif /* _CTHREADS_ */
 
+/* Hide the definitions which are only supposed to be used inside libc in
+   a separate file.  This file is not present in the installation!  */
+#ifdef _LIBC
+# include <bits/libc-lockP.h>
+#endif
+
 #endif	/* bits/libc-lock.h */
